### ISSUE TITLE
Europa requested fixes

### ISSUE THREAD

**Issue by BlueHNT** [Simple-Station/Einstein-Engines#1379]
# Description

So... [Europa](https://github.com/Simple-Station/Einstein-Engines/pull/1152) is out and it's pretty neat, but as engineer it's a nightmare.

Gonna list 'em out and it's gonna be a long one.

1. The SM Issues
So beginning with the whole issue that the passive cooling is useless is one thing as the planet has roughly 80-90°C and that won't work for the passive cooling AT ALL. I can manually fix it by closing it off with a buncha heaters, but normal people won't even notice it.
![image](https://github.com/user-attachments/assets/c9c89347-05cf-4031-a01a-45adaa359ef7)

On top of that, all engineering equipment is hidden in the SM room which is a bloody nightmare. You gotta pass by and lose your birthrights to even get your hardsuit.
![image](https://github.com/user-attachments/assets/eeac31b1-b6cf-4397-8e24-676892312f31)

Not to mention the nonsensical lockup of random equipment that doesn't even matter and a few tesla coils.
![image](https://github.com/user-attachments/assets/a6f5ac23-2dd3-49a3-b194-b8ab068621dc)

2. The Atmos and their issues
Oh boy the atmos, first of all it comes prepared already in it's tight packaged corner while at the same time missing all the features a regular atmos would need for this size of map. It even comes pre-made in a mega simple way, which means 90% of Atmos techs will just run some air and fuck off.
![image](https://github.com/user-attachments/assets/7df7e386-5d33-4809-aba5-cdee693587f5)

Next up the miner for Plasma and N2O is outright missing! Meaning you can't make tritium for engine, build TEG or make any normal SM mix.
![image](https://github.com/user-attachments/assets/1a07c344-d396-4ca2-b12c-1bafdb0889d3)

The little place for dedicated SM use is cute however.
![image](https://github.com/user-attachments/assets/afae6949-9014-462a-8880-aa5b0414340c)

Oh and roundstart the mixer chamber is missing Blast Doors to outside to vent your used mixes, while also missing the button for the already existing Blastdoor which however features a gracious tinyfan below it. 
![image](https://github.com/user-attachments/assets/aa29c12c-5d57-49f7-8341-28b0dc582aa9)

Additionally the map is also missing atmos techs hardsuits, meaning they can't fight fire even if they wanted with the non-pressure resistant fire suits they got.
![image](https://github.com/user-attachments/assets/63b37a2e-0578-4dbc-9981-0f38ee7e1dc9)

3. Tcomms?
Another issue I can see brewing is that all of engineering has full access to the TCOMMs which is bound to make no issues at all when a single antag in the department happens.
![image](https://github.com/user-attachments/assets/ee977e3c-8a73-4aac-94f1-e0f18bd51b20)

4. Impossible backup power
So it seems you chose material generators as a backup for the station, which makes it near impossible to run if a first time engineer tries booting up SM or forgoes it entirely as they don't understand what they're doing. At least the mats are generous?
![image](https://github.com/user-attachments/assets/6a68c95b-08f7-4342-875b-127a394cd793)

You could alternativelly expand on atmosia and give them a TEG.

Additonal notes:

There is definitely more issues than I just listed, but this is top of my head all that I could find in the span of this one round I went and tested things.

Off-note, what's happening with the whole permabrig being hidden behind sec right next to clown's room? The heck?

**Comment by BlueHNT** [comment]
Just tried to log my character off and couldn't find cryo-room... Is there... None?!

**Comment by Merrokitsune** [comment]
Did you try setting the spawn settings to “cryopods” and see?

**Comment by BlueHNT** [comment]
> Did you try setting the spawn settings to “cryopods” and see?

Yes, not only did I end up NOT on the station. I also then went into adminghost and tried searching for them manually, but the map has no markers...
![image](https://github.com/user-attachments/assets/7dc6fd73-41a9-4d19-b6dd-39aba955df36)

**Comment by sleepyyapril** [comment]
> Just tried to log my character off and couldn't find cryo-room... Is there... None?!

There is no cryo and I'll be removing Europa from rotation until this is solved, if TCJ agrees.

**Comment by BlueHNT** [comment]
Europa has the potential, but it has a lot of kinks atm.

**Comment by BlueHNT** [comment]
Turns out there's also plenty of rooms without air vents like in command? It might be a safety measure, but it does kind of ruin the room after 30 minutes in the shift.

**Comment by Mike32oz** [comment]
Let me do a quick check

**Comment by BlueHNT** [comment]
Atmos techs are also missing the piping network computer which is on Goobstation and similar, would really help this tbh.

**Comment by Mike32oz** [comment]
fun both air/waste distro are connected together.

**Comment by Mike32oz** [comment]
APC missing at Sophia/oracle 
![Screenshot 2024-12-28 055208](https://github.com/user-attachments/assets/6268f899-b803-49d3-8b14-cf0b5c6659a9)

**Comment by Mike32oz** [comment]
Mantis room has no air alarm/APC and no pipe air/waste
![Screenshot 2024-12-28 055435](https://github.com/user-attachments/assets/af16b206-39aa-446f-a0e5-5593f79a36d2)

**Comment by Mike32oz** [comment]
found cryogenic at  prison 
![Screenshot 2024-12-28 060148](https://github.com/user-attachments/assets/cd88528a-9935-44ab-b97b-cf6eddef8891)

**Comment by Ichaie** [comment]
The map is full of errors and issues, this is largely due to it being a very old map, which had several errors that were not corrected when I ported it, the rework I did was more to be temporary until I create a better layout in SM, there were some things missing but I can fix it very quickly, but not now because I'm focusing on another map currently unrelated to EE.
